docs/howtos/other/certificate_issuers.md

```diff
@@ -6,21 +6,22 @@ title: ""
 
 # Using a Certificate Issuer
 
-Epinio comes with multiple [cert-manager cluster issuers](https://cert-manager.io/docs/configuration/) for creating certificates:
+Epinio comes with multiple [cert-manager ClusterIssuer](https://cert-manager.io/docs/configuration/) for creating certificates.  
+By default it installs the `epinio-ca` and the `selfsigned-issuer`. If specified it will install also the `letsencrypt-staging` or `letsencrypt-production` issuer, to create certificates issued from [Let's Encrypt](https://letsencrypt.org/).
 
 * epinio-ca (default)
-* letsencrypt-production
-* selfsigned-issuer
+* selfsigned-issuer (internal)
+* letsencrypt-[staging|production] (optional)
 
-The issuer will be used for both, the Epinio API endpoint and workloads (i.e. pushed applications).
+The specified issuer will be used for both, the Epinio API endpoint and workloads (i.e. pushed applications).
 
 ## Choosing a Different Issuer
 
 When [installing Epinio with helm](../../installation/install_epinio.md#install-epinio), you can choose between those issuers by using the `global.tlsIssuer` helm variable.
 
-It's also possible to create a cert-manager cluster issuer in the cluster, before installing Epinio and referencing it by name when installing.
+It's also possible to create a cert-manager cluster issuer in the cluster, before installing Epinio and referencing it by name when installing, using the `global.customTlsIssuer`.
 
-When using the `letsencrypt-production` issuer further use the `global.tlsIssuerEmail` helm variable
+When using the `letsencrypt-staging` or `letsencrypt-production` issuer further use the `global.tlsIssuerEmail` helm variable
 to set an email address for the reception of the certificate notification emails sent by that
 issuer. __Note__, the default address is `epinio@suse.com`.
```

Comment:
Is that default still true ? I thought to have seen you remove that default in the chart PR.

Reply:
Ups, that's correct! I missed the note part. Thanks.

Reply:
Fixed with 266ccf7b1a182c35f8051ee2dc7921fe0d0f1518